bluemira/geometry/placement.py

```diff
@@ -99,10 +100,29 @@ def from_matrix(cls, matrix, label=""):
         obj.label = label
         return obj
 
+    @classmethod
```

Comment:
Modified in [1fcf7fd]. I have decided to declare such constants into placement.py (they could also be declared into __init__.p but since we still didn't agree on the external user interface through the __init__ file, I just avoided that).